### PR TITLE
Incremental synapse pipelines

### PR DESCRIPTION
**Pull request by jcbendernh**
Adding New Hackathon content of Incremental Synapse Pipelines.

### CODE REVIEW

File: 048-IncrementalSynapsePipelines/Coach/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# What The Hack - Incremental Synapse Pipelines <br> <i>Utilizing Change Data Capture in Azure SQL</i> <br> Coach's Guide 
+
+## Introduction
+Welcome to the coach's guide for the Automating Incremental Pipelines What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+Also remember that this hack includes a optional [lecture presentation](Automating-Incremental-Pipelines.pptx?raw=true) that features key topics for this What the Hack. 
+
+## Solutions
+- Challenge 0: **[Setup the Source and Target Environments](Solution-00.md)**
```
Comment (jrzyshr) [inline 048-IncrementalSynapsePipelines/Coach/README.md:9]: This is a silly title name. Can you do better?

**Review by jrzyshr — CHANGES_REQUESTED** [review]

File: 048-IncrementalSynapsePipelines/Coach/README.md (hunk 1 above)
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/Coach/README.md:9]: No I cannot.

Comment (jrzyshr) [reply 048-IncrementalSynapsePipelines/Coach/README.md:9]: Fine.

File: 048-IncrementalSynapsePipelines/Coach/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Session Guidance
+This What the Hack was created during the pandemic and was designed and executed for a remote/virtual audience.
+
+We initially conducted this hackathon in three - 3 hour sessions for a total of 9 hours.  It was done over a three week period with each session being conducted on a Friday.
+
+Moving forward, I would highly recommend that you dedicate atleast 12 hours overall to this hackathon.  This excludes breaks.  Thus, you may want to break this into 3 sessions that are a total of 5-6 hours long, including breaks. 
+
+Also, the feedback that we received from attendees is that they prefer that these sessions were held on consecutive days.  Since we did it every Friday, attendees felt that they spent too much time refershing their memory on what was accomplished the week(s) before.
+
+Challenge 2 will take the longest.  it will probably equal the length of challenge 0 and 1 put together.  Thus, please plan accordingly.
```
Comment (jordanbean-msft) [inline 048-IncrementalSynapsePipelines/Coach/README.md:27]: typo "It"

File: 048-IncrementalSynapsePipelines/Coach/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+   - Create the trigger within Synapse to automate the pipeline and add data to SQL and watch it flow through the staging and production tables in the dedicated pool and viewable in your Power BI Report.
+
+## Session Guidance
+This What the Hack was created during the pandemic and was designed and executed for a remote/virtual audience.
+
+We initially conducted this hackathon in three - 3 hour sessions for a total of 9 hours.  It was done over a three week period with each session being conducted on a Friday.
+
+Moving forward, I would highly recommend that you dedicate atleast 12 hours overall to this hackathon.  This excludes breaks.  Thus, you may want to break this into 3 sessions that are a total of 5-6 hours long, including breaks. 
+
+Also, the feedback that we received from attendees is that they prefer that these sessions were held on consecutive days.  Since we did it every Friday, attendees felt that they spent too much time refershing their memory on what was accomplished the week(s) before.
```
Comment (jordanbean-msft) [inline 048-IncrementalSynapsePipelines/Coach/README.md:25]: typo, "refreshing"

File: 048-IncrementalSynapsePipelines/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Challenge 1: **[Initial Data Load into the Dedicated Pool](Student/Challenge-01.md)**
+   - Use Synapse Pipelines to perform the initial data load
+- Challenge 2: **[Create Incremental Load Pipelines](Student/Challenge-02.md)**
+   - Implement Change Data Capture, create the synapse pipelines and the proper Dedicated Pool architecture to be used as a target for the pipelines and a source for reporting.
+- Challenge 3: **[Setup the Trigger to Automate the Incremental Load](Student/Challenge-03.md)**
+   - Create the trigger within Synapse to automate the pipeline and add data to SQL and watch it flow through the staging and production tables in the dedicated pool and viewable in your Power BI Report.
+
+## Prerequisites
+- One Azure Subscription per group.  One person must be the owner and the others the contributors.
+- Tool(s) to interact with Azure SQL and a Synapse Dedicated Pool.  Plenty of options here.
```
Comment (jordanbean-msft) [inline 048-IncrementalSynapsePipelines/README.md:30]: should you list some common tools such as SSMS, the Azure portal, etc?

File: 048-IncrementalSynapsePipelines/Student/Challenge-00.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# Challenge 0 - Setup the source and target environments
+
+**[Home](../README.md)** - [Next Challenge>](Challenge-01.md)
+
+## Introduction
+
+We need to setup the proper environment for the Hackathon.  Thus, we need everyone on the team to have access to the Azure SQL and Synapse Environments.  Also, any ancilliaries resources such as the Azure Storage Account and Key Vault.  
+
+## Description
```
Comment (jordanbean-msft) [inline 048-IncrementalSynapsePipelines/Student/Challenge-00.md:9]: should you provide ARM templates as a starting point?

File: 048-IncrementalSynapsePipelines/Coach/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+   - Implement Change Data Capture, create the synapse pipelines and the proper Dedicated Pool architecture to be used as a target for the pipelines and a source for reporting.
+- Challenge 3: **[Setup the Trigger to Automate the Incremental Load](Solution-03.md)**
+   - Create the trigger within Synapse to automate the pipeline and add data to SQL and watch it flow through the staging and production tables in the dedicated pool and viewable in your Power BI Report.
+
+## Session Guidance
+This What the Hack was created during the pandemic and was designed and executed for a remote/virtual audience.
+
+We initially conducted this hackathon in three - 3 hour sessions for a total of 9 hours.  It was done over a three week period with each session being conducted on a Friday.
+
+Moving forward, I would highly recommend that you dedicate atleast 12 hours overall to this hackathon.  This excludes breaks.  Thus, you may want to break this into 3 sessions that are a total of 5-6 hours long, including breaks.
```
[the remark this answers is not in the document]
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/Coach/README.md:23]: 👍.  Change made.

File: 048-IncrementalSynapsePipelines/Coach/README.md (hunk 2 above)
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/Coach/README.md]: 👍  Fixed

File: 048-IncrementalSynapsePipelines/Coach/README.md (hunk 3 above)
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/Coach/README.md]: 👍  Fixed

File: 048-IncrementalSynapsePipelines/Coach/Solution-01.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Challenge 1: Initial Data Load into the Dedicated Pool - Coach's Guide
+
+[< Previous Challenge](Solution-00.md) - **[Home](README.md)** - [Next Challenge>](./Solution-02.md)
+
+## Notes & Guidance
+For this challenge it is up to the student to determine how they want to initially copy the data over.
+
+We do not want them to over-engineer this component.  The key is to get the data over and if the distribution of the table is not correct in the Dedicated Pool, they can use CTAS to copy tables / create the correct distribution.  
+
+Many of these concepts are listed in the Learning Resources of the Student Guide.
```
[the remark this answers is not in the document]
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/Coach/Solution-01.md]: Good catch, I just added the hyperlink and successfully tested it.

File: 048-IncrementalSynapsePipelines/README.md (hunk 4 above)
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/README.md:30]: I did not want to list the tools as there are a variety of them that they can use like the ones you listed above but could also be Azure Data Studio, VS & VS Code.  I will leave it up to them to select their favorite.

File: 048-IncrementalSynapsePipelines/Student/Challenge-00.md (hunk 5 above)
Comment (jcbendernh) [reply 048-IncrementalSynapsePipelines/Student/Challenge-00.md:9]: I agree in a real environment with Dev/Test/Production that this should be ARM templated but for this exercise I would rather them talk it through and understand how to build this out manually and discuss all the items needed and how to configure them before getting into ARM templates.  Hope that makes sense.

**Review by jordanbean-msft — APPROVED** [review]

**Review by jrzyshr — APPROVED** [review]
Great addition to the WTH collection!  Thanks Jack & Chris for all of your efforts on this!

**Pull request merged**